src/containers/MainContainer/MainContainer.js

```diff
@@ -48,11 +49,9 @@ const MainContainer = () => {
     };
 
     useEffect(() => {
-        axios
-            .get('http://localhost:5000/api/questionnaires')
-            .then((response) => {
-                setQuestions(response.data.responses[0].questions);
-            });
+        axios.get(getQuestions).then((response) => {
```

Comment:
are you not able to use sendRequest? https://github.com/codeforsanjose/e-immigrate/blob/master/src/sendRequest/sendRequest.js

Reply:
Just updated to use sendRequest!